### PR TITLE
{bio}[GCCcore/11.2.0,foss/2021b] GetOrganelle v1.7.5.3, Bandage v0.9.0 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3510.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/96a925aa12f1ce0be6250099f40e6f62 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in total)
node3120.skitty.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/95a70e1804b6aac65b8a44eb5eab5848 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (2 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/f48227f382603158148bb234cd4bf3b0 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/1ef7df30d78039f82060d65794ec5e9d for a full test report.

**Comment by bedroge** [comment]
```
WARNING URL https://github.com/Kinggerm/GetOrganelle/archive/v1.7.5.3.tar.gz was not found (HTTP response code 404)
```

According to https://github.com/Kinggerm/GetOrganelle/releases, version 1.7.5.3 was released a few hours ago... Did they do a rerelease of this version?

File: easybuild/easyconfigs/g/GetOrganelle/GetOrganelle-1.7.5.3-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+name = 'GetOrganelle'
+version = '1.7.5.3'
+
+homepage = 'https://github.com/Kinggerm/GetOrganelle'
+description = """This toolkit assemblies organelle genome from genomic skimming data."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/Kinggerm/GetOrganelle/archive/']
+sources = ['v%(version)s.tar.gz']
```
Comment (bedroge) [inline easybuild/easyconfigs/g/GetOrganelle/GetOrganelle-1.7.5.3-foss-2021b.eb:12]: https://github.com/Kinggerm/GetOrganelle/commits/master
@boegel: looks like they've indeed re-released this version, and removed the `v` from the name of the tarball.

```suggestion
sources = ['%(version)s.tar.gz']
```

Comment (boegel) [reply easybuild/easyconfigs/g/GetOrganelle/GetOrganelle-1.7.5.3-foss-2021b.eb]: Indeed, and it seems like that's the only thing that has changed, the checksum is exactly the same as it was for `v1.7.5.3.tar.gz`, so I guess they just fixed the tag name they were using (from `v1.7.5.3` to `1.7.5.3`), to be in line with previous releases (cfr. https://github.com/Kinggerm/GetOrganelle/tags)

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--force-download"

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/edf5cac66319d4f4ab9ef472f484341f for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Pull request merged**